ipv8/loader.py

```diff
@@ -160,6 +181,15 @@ class A(CommunityLauncher):
         class A(CommunityLauncher):
            ...
 
+       def my_strategy_class():
+            from my_module.some_submodule import MyStrategyClass
+            return MyStrategyClass
+           ...
+
+        @walk_strategy(my_strategy_class)
+        class A(CommunityLauncher):
+            pass
```

Comment:
Looking good 👍 , please don't forget the second half of the request for changes: consistent use of the `...`:

```suggestion
       def my_strategy_class():
            from my_module.some_submodule import MyStrategyClass
            return MyStrategyClass

        @walk_strategy(my_strategy_class)
        class A(CommunityLauncher):
            ...
```

After that, please squash your commits (please contact me if you need help and I'll give you the appropriate git commands - this should take about 30 seconds).

Then, finally, this should be completely done.